source/_patterns/02-molecules/vue-widgets/_vue-widgets.scss

```diff
@@ -2,5 +2,5 @@
 // are provided to this file via:
```

Comment:
This component folder should be called `vue-widget` (singular). `vue-widget/demo/vue-widgets.twig` will be plural. It's a convention all components follow.